tools/protagonist.js

```diff
@@ -8,8 +8,8 @@ var protagonist = require('../build/Release/protagonist');
 //var protagonist = require('protagonist');
 
 // Add Segfault handler
-var segfaultHandler = require('segfault-handler');
-segfaultHandler.registerHandler();
+// var segfaultHandler = require('segfault-handler');
+// segfaultHandler.registerHandler();
```

Comment:
I don't understand why these are removed. If you want to remove them completely, please remove them from package.json too.